### DISCUSSION TITLE
Standalone collector mode - G7

### DISCUSSION THREAD

**Discussion by barth11** [NightscoutFoundation/xDrip#3394] [Q&A]
Hi,
I want to buy Galaxy Watch 6 but I'd like to use G7 with watch directly using Standalone collector mode. As of now, it's now listed but before I buy the watch, can you suggest if I need the newest and the mode will be supported. As far as I read [this](https://xdrip.readthedocs.io/en/latest/smartwatch/wear/#standalone-collector-mode) the mode is limited.
I hope I ask at the right place :)
Thanks.
B.

**Comment by Navid200** [comment]
As far as I know, no watch can directly collect from G7.
Dexcom has announced that apple watch will be able to do that some time in the future.  But, I am not certain.
